docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+# WebRTC
+
+You can have real-time audio/video communication with a remote peer. You can send and receive multimedia sources and generic data with a remote peer. The multimedia sources include audio/video stream from microphone, camera or media file. The generic data includes string or byte data.
+
+The main features of the WebRTC API include:
```

Comment:
Lists should be introduced with a complete sentence.
Ref: https://developers.google.com/style/lists#introductory-sentences-for-lists
```suggestion
The main features of the WebRTC API include the following:
```